console/console-init/ui/src/modules/iot-device/dialogs/CreateDevice/CreateDevicePage.tsx

```diff
@@ -85,13 +97,18 @@ export default function CreateDevicePage() {
 
   const reviewForm = {
     name: "Review",
-    component: <p>Review</p>
+    component: (
+      <ReviewDeviceContainer
+        device={device}
+        title={"Verify that the followwing information is correct before done"}
+      />
+    )
   };
 
   const onConnectionChange = (_: boolean, event: any) => {
-    const data = event.target.value;
-    if (data) {
-      setConnectionType(data);
+    const connectionType = event.target.value;
+    if (connectionType) {
+      setDevice({ ...device, connectionType: connectionType });
```

Comment:
don't need to use connectionType two time, both are same.
 setDevice({ ...device, connectionType });